src/components/date/__internal__/date-picker/date-picker.spec.tsx

```diff
@@ -221,7 +263,7 @@ describe("StyledDayPicker", () => {
       });
     });
 
-    describe.each([
+    describe.each<keyof typeof translations>([
```

Comment:
```suggestion
    describe.each([
```

**suggestion** (non-blocking): as suggested elsewhere, this isn't necessary as long as you put an `as const` after the array